src/components/Firestore/DocumentPreview/index.test.tsx

```diff
@@ -306,7 +306,7 @@ describe('loaded map', () => {
     expect(getByText('first_name')).not.toBe(null);
     expect(getByText('last_name')).not.toBe(null);
 
-    getByText('{last_name: "potter", first_name: "harry"}').click();
+    getByText('{first_name: "harry", last_name: "potter"}').click();
```

Comment:
note: This change is good enough for test coverage. There is no need to add another test case for now.

Reply:
Great! Thanks